src/FlaUI.WebDriver.UITests/SessionTests.cs

```diff
@@ -83,25 +84,26 @@ public void NewSession_AppNotAString_Throws(object value)
         public void NewSession_AppWorkingDir_IsSupported()
         {
             var driverOptions = FlaUIDriverOptions.TestApp();
-            driverOptions.AddAdditionalOption("appium:appWorkingDir", "C:\\");
-            using var driver = new RemoteWebDriver(WebDriverFixture.WebDriverUrl, driverOptions);
+            driverOptions.AddAdditionalAppiumOption("appium:appWorkingDir", "C:\\");
+            using var driver = new WindowsDriver(WebDriverFixture.WebDriverUrl, driverOptions);
 
             var title = driver.Title;
 
             Assert.That(title, Is.EqualTo("FlaUI WPF Test App"));
         }
 
         [Test]
+        [Ignore("AppiumOptions prefixes timeouts with 'appium:' prefix. Not sure what to do here.")]
```

Comment:
Note sure what to do here? Maybe `timeouts` should have the prefix, maybe it shouldn't? I'm not familiar enough with this area to say.

Reply:
I would like to keep support for the WebDriver2 `"timeouts"` capability (see WebDriver2 capabilities [here](https://www.w3.org/TR/webdriver2/#capabilities)). The Appium standard [does not describe](https://appium.io/docs/en/2.0/guides/caps/) an `"appium:timeouts"` capability, so prefixing it is not valid.

Can't we use the named properties `ImplicitWaitTimeout`, `PageLoadTimeout`, `ScriptTimeout` instead?